ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -436,3 +436,18 @@ class BaremetalRedfishIPxeWholediskHttpLink(
     @utils.services('network')
     def test_ip_access_to_server(self):
         self.boot_and_verify_node()
+
+class BaremetalIdracDirectWholediskHttpLink(
+        bsm.BaremetalStandaloneScenerioTest):
+
+    api_microversion = '1.31'
+    driver = 'idrac'
+    deploy_interface = 'direct'
+    boot_interface = 'idrac-redfish-virtual-media'
```

Comment:
**boot_interface** should be a mandatory attribute. Take reference of previous test cases raised (Management and Bios)

Reply:
done